lib/foreman_tasks/tasks/export_tasks.rake

```diff
@@ -216,11 +216,22 @@ namespace :foreman_tasks do
         end
       end
 
+      def self.count_duration(task)
+        if task.started_at && task.ended_at
+          (task.ended_at - task.started_at).floor
+        elsif task.started_at && task.ended_at.nil?
+          (Time.now.to_i - task.started_at).floor
```

Comment:
If both `started_at` and `ended_at` are non-nil, you just subtract them, but if the duration is open ended, you convert the time to integer first. Why?

Reply:
It should be `to_f` not `to_i`

Reply:
But the question remains. Why do you subtract two times in one branch, but time from integer/float in another?

Reply:
I added there `to_i` then `to_f`, because of rubocop testing... so should be there `to_f` for each time?

Reply:
What exactly was rubocop complaining about?

Reply:
```
lib/foreman_tasks/tasks/export_tasks.rake:223:17: C: Rails/TimeZone: Do not use Time.now without zone. Use one of Time.zone.now, Time.current, Time.now.in_time_zone, Time.now.utc, Time.now.getlocal, Time.now.xmlschema, Time.now.iso8601, Time.now.jisx0301, Time.now.rfc3339, Time.now.httpdate, Time.now.to_i, Time.now.to_f instead.
```